Guided Investigation - Incident Triage.ipynb

```diff
@@ -57,40 +57,28 @@
    },
```

Comment:
<p>&lt;details&gt;?</p><p>Is there a title?</p><p> </p><p>At some point it would be cool to turn this into a notebooklet (at some yet-to-be-specified future date)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>